test/integration/Command/InstallCommandTest.php

```diff
@@ -62,6 +64,44 @@ public function testInstallCommandWillInstallCompatibleExtension(): void
             return;
         }
 
-        (new Process(['sudo', 'rm', $matches[1]]))->mustRun();
+        $fileToRemove = $matches[1];
+        assert(is_string($fileToRemove));
+        $rmCommand = ['rm', $fileToRemove];
+        if (! is_writable($fileToRemove)) {
+            array_unshift($rmCommand, 'sudo');
+        }
+
+        (new Process($rmCommand))->mustRun();
+    }
+
+    public function testInstallCommandWillInstallCompatibleExtensionWindows(): void
+    {
+        if (PHP_VERSION_ID < 80300 || PHP_VERSION_ID >= 80400) {
+            self::markTestSkipped('This test can only run on PHP 8.3 - you are running ' . PHP_VERSION);
```

Comment:
Why is this PHP 8.3 only?

Reply:
Testing is tricky unless you have multiple versions of PHP available (e.g. using deb.sury.org, or you compile several different versions of PHP); therefore to reduce reliance on that particular complexity, this test only runs in PHP 8.3. Can certainly be worked on, but it will make CI much more complex :sweat_smile: